pkg/sql/logictest/testdata/logic_test/create_as

```diff
@@ -382,3 +382,10 @@ query I
 SELECT * FROM tab_from_seq
 ----
 2
+
+# Regression test for #81554, where tried to do gigantic batches for CTAS in
+# explicit transactions
+statement ok
+BEGIN;
+CREATE TABLE source_tbl_huge AS SELECT 1::CHAR(256) FROM generate_series(1, 500000);
```

Comment:
I was curious and tried this on `master` and got
```
ERROR: command is too large: 126737600 bytes (max: 67108864)
```

Perhaps I'm too paranoid but those two values are too close for comfort. I'd be more comfortable if we explicitly set the `kv.raft.command.max_size` setting for this test to a specific value (like the floor value of 4MB) and explained why we're doing so in the comment for this test. The test itself is great otherwise.